wp-content/themes/pawar2018/front-page.php

```diff
@@ -1,98 +1,88 @@
-<?php get_header(); ?>
+<?php
+/*
+Template name: Front Page Template
+ */
+get_header(); ?>
   <div class="promo-banner">
     <h1 class="banner-headline">
       Join Ameya at the June 15th Virtual Town Hall.
       <a href="http://watchparty.pawar2018.com" target="_blank">RSVP Now</a>
     </h1>
   </div>
   <main>
-    <section class="hero">
+    <section class="hero" style="background-image: url('<?php the_field('hero_image'); ?>');">
       <div class="row align-bottom">
         <div class="small-11 large-9 columns">
-          <h1>A New Deal For Illinois</h1>
+          <h1><?php the_field('hero_headline'); ?></h1>
         </div>
       </div>
       <div class="row align-middle">
         <div class="small-11 large-7 columns">
-          <h2>It‘s time to grab a clipboard, lace up our boots, and work together to restore progressive values in 2018.</h2>
+          <h2><?php the_field('hero_subheadline'); ?></h2>
         </div>
       </div>
       <div class="row align-middle">
         <div class="small-11 columns">
-          <h3>Get the latest updates</h3>
-          <a href="/newsletter" class="button">Subscribe</a>
+          <h3><?php the_field('hero_cta'); ?></h3>
+          <a href="<?php the_field('hero_button_link'); ?>" class="button">
+            <?php the_field('hero_button_text'); ?>
+          </a>
         </div>
       </div>
     </section>
     <section class="row main-content align-center align-middle">
+      <?php if (get_field('has_video')) : ?>
         <div class="small-11 large-6 columns">
           <div class="flex-video widescreen">
-            <iframe width="500" height="282" src="https://www.youtube.com/embed/0QN_PvW9nQw?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+            <?php the_field('featured_video'); ?>
```

Comment:
Should the contents of `featured_video` be just a link to the video URL? It appeared that way from the form, but after dropping a link to the youtube URL, it doesn't seem to be appearing.